components/chef-ui-library/src/global/variables.css

```diff
@@ -27,6 +27,7 @@
   --blue-de-france: #3DA5FF;
   --gray: #B7BCBC;
   --blue: #174AF0;
+  --palatinate-blue: #0E3DD5;
```

Comment:
Wondering what your source is for this color?
Every reference I can find (e.g. https://encycolorpedia.com/search?q=palatinate+blue) shows palatinate blue as `#273be2`

Reply:
I got if from the Automate Extended Color Palette.